### ISSUE TITLE
ISSN 2366-0597 | Forum Exegese und Hochschuldidaktik: Verstehen von Anfang an (VvAa) | Abstract

### ISSUE THREAD

**Issue by kdr154** [ubtue/DatenProbleme#1241]
Abstracts gibt es in Deutsch und auf Englisch. Übertragen wird aber nur eines.
https://elibrary.narr.digital/article/10.2357/VvAa-2020-0003

**Comment by socheres** [comment]
welches Translator haben Sie verwendet?

**Comment by helenanebel** [comment]
Lieber @kdr154 ,
ich habe einen neuen Translator ubtue_Narr erstellt, der das Abstract und auch die Keywords mitnimmt.
Viele Grüße, Helena Nebel

**Comment by kdr154** [comment]
Ich erhalte weiterhin nur das deutsche Abstract und die Keywords werden auch nicht übertragen. 
Nebenbei: Inzwischen ist dies eine Kooperationszeitschrift und sie soll von "Bildi", den Innsbruckern erfasst werden. Da die auch mit Zotero arbeiten ist es in diesem Fall wichtig, dass die den aktualisierten Translator auch erhalten. Ansprechpartner auf unserer Seite ist m. W. @IxKo

**Comment by IxKo** [comment]
Beide Abstracts werden nun übertragen.

Wäre es eventuell möglich, dass die Rezensionen erkannt werden?

Und bei diesem Aufsatz wird am Ende des Titels ein Doppelpunkt gesetzt, obwohl dort keiner ist:
https://elibrary.narr.digital/article/10.24053/VvAa-2020-0020
![image](https://user-images.githubusercontent.com/57000004/141977180-1277b6ad-bdc0-4cf5-85e5-f0bc7add4a71.png)

**Comment by IxKo** [comment]
Der Doppelpunkt am Ende wird nun nicht mehr erzeugt.
Die Rezensionen werde nicht automatisch erkannt. Schade, wahrscheinlich nicht möglich.

**Comment by socheres** [comment]
Rezensionstag wird gesetzt. Z.B. https://elibrary.narr.digital/article/10.24053/VvAa-2020-0018

**Comment by socheres** [comment]
Voraussetzung dafür ist das Vorhandensein einer ISBN im Titel.

**Comment by IxKo** [comment]
Es wird das Tag "Book Review" gesetzt. Im halbautomatischen Verfahren müsste aber RezensionstagPica gesetzt werden.

**Comment by socheres** [comment]
Jetzt müsste auch mit "Book Review" funktionieren.

**Comment by IxKo** [comment]
Ja, funktioniert nun auch mit "Book Review". Könnte dann nicht das "RezensionstagPica" zu Book Review geändert werden? Sonst hat es zwei Tags, die ein und dasselbe machen.
